pkg/server/k8s/client.go

```diff
@@ -784,6 +796,35 @@ func (c *Client) patchCronJobEnvVars(namespace, name string, v interface{}) erro
 	return errors.Wrap(err, "patch cronjob failed")
 }
 
+func convertAppSecretEnvVar(secretName string, secrets []string) interface{} {
```

Comment:
Maybe put this on `converters.go`

Reply:
I'm not sure, `converters.go` is for _teresa struct_ -> _k8s struct_